### ISSUE TITLE
All writemarkers are failing on dev n/w

### ISSUE THREAD

**Issue by devyetii** [0chain/0chain#2349]
All write markers are failing in dev n/w. The following error appears.
```
2023-04-28 17:44:34	
2023-04-28T13:50:10.134Z	DEBUG	chain/state.go:393	Error executing the SC, chargeable error	{"error": "commit_connection_failed: item not found: 5c61f8d3e63528dfe45db89a598bd5c42c71b3994f7639d4647268ba75269d9a", "client id": "5c61f8d3e63528dfe45db89a598bd5c42c71b3994f7639d4647268ba75269d9a", "block": "25d3a945dbb3b738038d2a1e5158338c1948b3ba1b6fa433e5806d6d9404b8fb", "begin client state": "f58afacd377fb156af519468070b8e868a412e9feaf0f7459e6775f04a4bdd15", "prev block": "deef9d14e4f4cccf28f51d90ce4de9bd8b3ada4b6d1f1ce84512c8fe5fb6087a", "time_spent": "1.597621ms", "txn": {"hash":"4093bbd935a04f2bfbddb8d01cbbc4c646719307a8cfb7732421dcf2bebd6359","version":"1.0","client_id":"5c61f8d3e63528dfe45db89a598bd5c42c71b3994f7639d4647268ba75269d9a","public_key":"8087f9ef29fa55efc35af433b63f78125a892da7be90b396e80f5ad0fecaa40055312c31f1cbd261351f77ccbb54398a13244b1ec5c9aa47022461c0bd888a1f","to_client_id":"6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7","chain_id":"0afc093ffb509f059c55478bc1a60351cef7b4e9c008a53a6cc8241ca8617dfe","transaction_data":"{\"name\":\"commit_connection\",\"input\":{\"allocation_root\":\"8ce7ad392268e6ff0ea6b23f9a34d3433e513a5d9c6368928b17f8ddf7dcfdf2\",\"prev_allocation_root\":\"\",\"write_marker\":{\"allocation_root\":\"8ce7ad392268e6ff0ea6b23f9a34d3433e513a5d9c6368928b17f8ddf7dcfdf2\",\"prev_allocation_root\":\"\",\"file_meta_root\":\"42012470e20e90036b3098da71e1056ce0e561031bc41fa47faa1d1269f93a2e\",\"allocation_id\":\"4fa59b9b2a27c4eb910cbb9c96877f7d277ff29d2aec65f255a60001c8a30e65\",\"size\":0,\"blobber_id\":\"5c61f8d3e63528dfe45db89a598bd5c42c71b3994f7639d4647268ba75269d9a\",\"timestamp\":1682617084,\"client_id\":\"0cb40d3bb59ddbf4902a4c32b12ffe4ede569ab358d31fc144b90fbf0e37b7a7\",\"signature\":\"7dce1d1bd7b487bd175c7353035567d9d5e347eacaf1353d61bff7a7ce1e189a\"}}}","transaction_value":0,"signature":"aa9b4157e5f5bc66d5852ef74828539b7a07794737bae0e0492064e22be19ca0","creation_date":1682689809,"transaction_fee":1000000000,"transaction_nonce":10031,"transaction_type":1000,"transaction_output":"commit_connection_failed: item not found: 5c61f8d3e63528dfe45db89a598bd5c42c71b3994f7639d4647268ba75269d9a","txn_output_hash":"e9af38347c6127413668b38489ed22da2a64b8cf307562f70418e6c25d8fad88","transaction_status":2}} 
```
TLDR; `commit_connection_failed: item not found: 5c61f8d3e63528dfe45db89a598bd5c42c71b3994f7639d4647268ba75269d9a"`
Related to [this function](https://github.com/0chain/0chain/blob/fafae179f7dd975a9d2beb3a61f50331108efdd0/code/go/0chain.net/smartcontract/storagesc/challenge_ready_partition.go#L56-L68) in 0chain repo.
Related discussion: https://0chain.slack.com/archives/GREHVKQKS/p1682696274385199

**Comment by dabasov** [comment]
fixed with https://github.com/0chain/blobber/pull/1067
